### PR TITLE
Existing service account sr

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

**Review by wtrocki — APPROVED** [review]

File: getting-started-service-registry/README.adoc
Comment (jbyrne-redhat) [inline getting-started-service-registry/README.adoc:189]: Given the way quick starts are presented as a series of discrete tasks in the Hybrid Cloud Console, referring to _sections_ doesn't feel _quite_ right for that delivery format.

We could either:

- Split the difference between the cloud console and Customer Portal, and generically refer to them as _procedures_.
- Use conditionalization (`ifdef::qs[]`, ifndef::qs[]) to refer to _tasks_ in the cloud console, and _sections_ on the Customer Portal.

**Review by jbyrne-redhat — APPROVED** [review]
Looks good to me overall, @bredamc  - good work! I left one comment that we might consider, but it's not a big deal.

**Comment by bredamc** [comment]
Great suggestion, thanks @jbyrne-redhat -- I've used the conditional text approach that you suggested :)

**Review by smccarthy-ie — APPROVED** [review]
LGTM, thanks!

**Comment by bredamc** [comment]
@jbyrne-redhat @smccarthy-ie Can you please give one last sanity check before I merge? Many thanks :)

**Comment by jbyrne-redhat** [comment]
> @jbyrne-redhat @smccarthy-ie Can you please give one last sanity check before I merge? Many thanks :)

Looks good to me, @bredamc!

**Comment by bredamc** [comment]
@jbyrne-redhat Updated the Kafka doc to add the paragraph about an existing service account -- please review, and approve if happy! Thank you :)

**Review by jbyrne-redhat — APPROVED** [review]
Kafka additions look good too, @bredamc!

**Pull request merged**
